pod/perldelta.pod

```diff
@@ -5,35 +5,216 @@
+
+This allows you to declare that the portion of a program for the
+remainder of the lexical scope of this pragma is encoded either entirely
+in ASCII (for S<C<use source::encoding 'ascii'>>) or if UTF-8 is allowed
+as well (for S<C<use source::encoding 'utf8'>>). No other encodings are
+accepted. The second form is entirely equivalent to S<C<use utf8>>, and
+may be used interchangeably with that.
+
+The purpose of this pragma is to catch cases early where you forgot to
+specify S<C<use utf8>>.
+
+S<C<use source::encoding 'ascii'>> is automatically enabled within the
+lexical scope of a S<C<use v5.41.0>> or higher.
+
+S<C<no source::encoding>> turns off all this checking for the remainder of
+its lexical scope. The meaning of non-ASCII characters is then undefined.
+
+=head2 New C<:writer> attribute on field variables
+
+Classes defined using C<use feature 'class'> are now able to automatically
+create writer accessors for scalar fields, by using the C<:writer> attribute,
+similar to the way that C<:reader> already creates reader accessors.
+
+    class Point {
+        field $x :reader :writer :param;
+        field $y :reader :writer :param;
+    }
+
+    my $p = Point->new( x => 20, y => 40 );
+    $p->set_x(60);
+
+=head2 New C<any> and C<all> operators
+
+Two new experimental features have been added, which add two new list-processing
+operators, C<any> and C<all>.
+
+    use v5.40;
+    use feature 'keyword_all';
+    no warnings 'experimental::keyword_all';
+
+    my @numbers = ...
+
+    if(all { $_ % 2 == 0 } @numbers) {
+        say "All the numbers are even";
+    }
+
+These keywords operate similarly to C<grep> except that they only
+ever return true or false, testing if any (or all) of the elements in
+the list make the testing block yield true.  Because of this they can
+short-circuit, avoiding the need to test any further elements if a given
+element determines the eventual result.
+
+These are inspired by the same-named functions in the L<List::Util> module,
+except that they are implemented as direct core operators, and thus perform
+faster, and do not produce an additional subroutine call stack frame for
+invoking the code block.
+
+The feature flags enabling those keywords have been named L<keyword_any>
+and L<keyword_all> to avoid confusion with to the ability of the
+C<feature> module to refer to all of its features by using the C<:all>
+export tag.  [L<GH #23104|https://github.com/Perl/perl5/issues/23104>]
+
+The related experimental warning flags are consequently named
+C<experimental::keyword_any> and C<experimental::keyword_all>.
+
+=head2 Apostrophe as a global name separator can be disabled
+
+This was deprecated in Perl 5.38 and removed as scheduled in perl
+5.41.3, but after some discussion has been reinstated by default.
+
+This can be controlled with the C<apostrophe_as_package_separator>
+feature which is enabled by default, but is disabled from the 5.41
+feature bundle onwards.
```

Comment:
For this perldelta, should it read "disabled from the 5.42 feature bundle onwards"?

Reply:
I think so

Reply:
Technically, `:5.41` is an alias to `:5.42`. The documentation for `feature` also says: "This is enabled by default, but disabled from the 5.41 feature bundle
onwards."

However, the use of development bundles is not very consistent in `feature.pm`, so maybe we should just mention stable versions in there.